ui/de-webapp/src/main/java/org/iplantc/de/server/controllers/DeController.java

```diff
@@ -46,12 +49,30 @@ public String redirectToDe(final HttpServletRequest request) throws MalformedURL
         return "redirect:de/";
     }
 
+    private DiscoveryEnvironmentMaintenance getDeMaintenance() {
+        return new DiscoveryEnvironmentMaintenance(maintenanceFile);
```

Comment:
Yeah, `user.dir` resolves to the current working directory, so it's effectively the same as long as the path doesn't start with a path separator. When the path does start with a path separator, the current code will treat the path as absolute whereas the previous code will treat it as relative to the current working directory. The drop page code has always used just the maintenance file property name, so I went with that option.